pipelines/pipedrive_pipeline.py

```diff
@@ -30,6 +30,13 @@ def load_selected_data():
     print(pipedrive_data.deals_participants)
 
 
+def load_new_data():
+    """Example setting an initial value for incremental loading to get activities updated after given date"""
+    pipeline = dlt.pipeline(pipeline_name='pipedrive', destination='postgres', dataset_name='pipedrive_data')
+    load_info = pipeline.run(pipedrive_source(since_timestamp="2023-03-01 00:00:00Z").with_resources("activities", "custom_fields_mapping"))
```

Comment:
I think the most common case is that we load everything from the beginning and limit just the activities. it should be demonstrated. We can call the `pipedrive_source` twice. in the first one we load from the beginning and then deselect activities. the second one is as above. you can pass a list of soures to a single `run` call
there's also a chance that `source.activities.apply_hints(incremental=...) will work

Reply:
Example for this https://github.com/dlt-hub/pipelines/pull/84/commits/f93c29509177667518789cad06facffa05ac33fc